File: src/main/scala/turing/utils/PropertiesUtils.scala
package turing.utils

import java.io.{IOException}
import java.util.Properties

object PropertiesUtils {

  def loadProperties(resourcePath: String): Properties = {
    val prop = new Properties
    new PropertiesUtilsCls().appendProperties(resourcePath, prop)
    prop
  }
}

class PropertiesUtilsCls {
  def appendProperties(fileName: String, prop: Properties): Unit = {
    val propStream = classOf[PropertiesUtilsCls].getClassLoader.getResourceAsStream(fileName)
    try
      prop.load(propStream)
    catch {
      case e: IOException =>
        e.printStackTrace()
    }
  }
}

File: src/main/java/javautils/PropertiesUtils.java
package javautils;

import java.io.IOException;
import java.io.InputStream;
import java.util.Properties;

public class PropertiesUtils {

    private static void appendProperties(String resourceFile, Properties properties){
        InputStream propStream = PropertiesUtils.class.getClassLoader().getResourceAsStream(resourceFile);
        try {
            properties.load(propStream);
        } catch (IOException e) {
            e.printStackTrace();
        }
    }

    public static Properties loadProperties(String resourcePath){
        Properties properties = new Properties();
        appendProperties(resourcePath, properties);
        return properties;
    }

}
